test/pytest/test_depthconv1d.py

```diff
@@ -21,18 +24,22 @@
 @pytest.mark.parametrize('kernels', kernel_options)
 @pytest.mark.parametrize('bias', bias_options)
 @pytest.mark.parametrize(
-    'backend, io_type',
+    'backend, io_type, strategy',
     [
-        ('Vivado', 'io_parallel'),
-        ('Vitis', 'io_parallel'),
-        ('Vivado', 'io_stream'),
-        ('Vitis', 'io_stream'),
-        ('Catapult', 'io_stream'),
+        # ('Vivado', 'io_parallel', 'latency'),
+        # ('Vitis', 'io_parallel', 'latency'),
+        # ('Vivado', 'io_stream', 'latency'),
+        # ('Vitis', 'io_stream', 'latency'),
+        ('Vivado', 'io_stream', 'resource'),
+        ('Vitis', 'io_stream', 'resource'),
+        # ('Catapult', 'io_stream', 'latency'),
```

Comment:
This is a debugging setup that got committed. We should uncomment the useful checkps